Cargo.toml

```diff
@@ -7,7 +7,7 @@ categories = ["cryptography"]
 homepage = "https://tari.com"
 readme = "README.md"
 license = "BSD-3-Clause"
-version = "0.13.2"
+version = "0.13.4"
```

Comment:
Don't update the version, this is done in the tagging process

Reply:
Ok cool